ui/src/org.ts

```diff
@@ -1,22 +1,54 @@
-import * as api from "./api"
-import { createValidationStore } from "./validation"
-import { Transaction } from './transaction';
-import { EmojiAvatar } from "./avatar"
+import { createValidationStore } from "./validation";
+import { EmojiAvatar } from "./avatar";
+import { Transaction } from "./transaction";
+
+import * as api from "./api";
+import * as event from "./event";
+import * as remote from "./remote";
+
+// TYPES.
+export interface Org {
+  name: string;
+  avatarFallback: EmojiAvatar;
+}
+
+interface Fetch extends event.Event<Kind> {
+  kind: Kind.Fetch;
+  id: string;
+}
+
+interface RegisterOrgInput {
+  id: string;
+}
+
+type Msg = Fetch;
 
 export enum RegistrationFlowState {
   NameSelection,
   TransactionConfirmation
 }
 
-export interface Org {
-  name: string;
-  avatarFallback: EmojiAvatar;
+// STATE
+const orgStore = remote.createStore<Org>();
+export const org = orgStore.readable;
+
+// EVENTS
+enum Kind {
```

Comment:
This should be defined before usage, thought our liniting catches that.

Reply:
It catches some of the ordering issues, but not all. Weird.